docs/guides/two_factor_authentication.md

```diff
@@ -0,0 +1,241 @@
+# Two-factor Authentication
+
+Two-factor authentication provides an extra layer of security for admin users by using, as the name suggests, an additional factor in the login process (a time-based one-time usage passcode, in addition to the usual email/password pair).
+
+This guide will provide a brief explanation to understand how the two-factor authentication system works in the eWallet.
+
+## Table of Contents
+
+- [Two-factor Authentication](#two-factor-authentication)
+  - [Table of Contents](#table-of-contents)
+    - [Enable 2FA](#enable-2fa)
+      - [Create a Secret Code](#create-a-secret-code)
+      - [Create Backup Codes](#create-backup-codes)
+    - [Log in with 2FA](#log-in-with-2fa)
+    - [Disable 2FA](#disable-2fa)
+    - [Note](#note)
+
+### Enable 2FA
+
+To enable two-factor authentication, a secret code must be provided. This secret code is generated through an endpoint of the eWallet. The section below will explain how to get one.
+
+#### Create a Secret Code
+
+The secret code is a unique 16 character alphanumeric code used to create the time-based one-time passcodes (each generated code will only be valid for x seconds).
+
+To create a secret code, call to `/me.create_secret_code` with the appropriate parameters:
+
+```bash
+curl http://localhost:4000/api/admin/me.create_secret_code \
+-X POST \
+-H "Accept: application/vnd.omisego.v1+json" \
+-H "Authorization: OMGAdmin $(echo -n "user_id:auth_token" | base64 | tr -d '\n')" \
+-H "Content-Type: application/json" \
+-d '{}' \
+-v -w "\n" | jq
+```
+
+The response will be similar to:
+
+```json
+{
+  "version": "1",
+  "success": true,
+  "data": {
+    "secret_2fa_code": "EDK4WLQ2A5DXYFAL",
+    "object": "secret_code",
+    "label": "user@example.com",
+    "issuer": "OmiseGO"
+  }
+}
+```
+
+We can now create a two-factor QR code by using the data received in this response.
+
+> Try: using a web service to quickly generate the Two-factor QR Code e.g. [this](https://stefansundin.github.io/2fa-qr/)
```

Comment:
```suggestion
**Try:** use a web service to quickly generate a two-factor QR code, e.g. [Stefan Sundin's 2FA QR code generator](https://stefansundin.github.io/2fa-qr/)
```

Let's keep quotes for actual quotes. Also link titles should be descriptive about the link for SEO purposes.